python/ray/tune/tuner.py

```diff
@@ -432,3 +432,12 @@ def get_results(self) -> ResultGrid:
                 string_queue,
             )
             return ray.get(get_results_future)
+
+    def __getattribute__(self, item):
+        if item == "restore":
```

Comment:
I wish there was a cleaner way to do this directly in the definition of `restore`, but I'm not sure if there is one...

Reply:
I don't think so, at least I can't think of one right now. The `@classmethod` decorator passes the class in both cases...